### PR TITLE
Group all non-major updates into a single PR

### PR DESCRIPTION
**Pull request by p-j-smith**
## Description
Renovate is busy, and using lots of resources in CI. Suggestion to group all non-major updates into a single PR

## Type of change
Please delete options accordingly to the description.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Suggested Checklist

- [ ] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have passed on my local host device. (see further details at the [CONTRIBUTING](https://github.com/SAFEHR-data/PIXL/blob/main/CONTRIBUTING.md#local-testing) document)
- [ ] Make sure your branch is up-to-date with main branch. See [CONTRIBUTING](https://github.com/SAFEHR-data/PIXL/blob/main/CONTRIBUTING.md) for a general example to syncronise your branch with the `main` branch.
- [ ] I have requested review to this PR.
- [ ] I have addressed and marked as resolved all the review comments in my PR.
- [ ] Finally, I have selected `squash and merge`

### CODE REVIEW

File: .renovaterc.json5 [hunk 1]
```diff
@@ -2,7 +2,8 @@
   $schema: "https://docs.renovatebot.com/renovate-schema.json",
   extends: [
     "github>UCL-ARC/.github//renovate/default-config.json",
-    "schedule:monthly",
```
Comment (p-j-smith) [inline .renovaterc.json5:5]: the UCL-ARC config already extends from `schedule:monthly`

Comment (stefpiatek) [reply .renovaterc.json5:5]: so what's interesting is that it isn't running on a schedule. I had a hunch it wasn't being ran, so went to the ui and made the mistake of selecting all thinking that we had set up the grouping. Not high up on our list of issues though

**Review by stefpiatek — APPROVED** [review]
🎉

File: .renovaterc.json5 (hunk 1 above)
Comment (p-j-smith) [reply .renovaterc.json5:5]: oh strange, maybe I'll take a quick look next month if it doesn't run automatically

**Pull request merged**